WeatherNotFound/src/main/java/me/learning/weathernotfound/presentation/WeatherNotFound.kt

```diff
@@ -105,6 +113,19 @@ class WeatherNotFound private constructor() {
         }
 
         validateOpenWeatherApiKey {}
+
+        Log.i(TAG, "initialized completely")
+    }
+
+    fun tryAutoInit(context: Context) {
+        val canAutoInit = context.packageManager
+            .getApplicationInfo(context.packageName, PackageManager.GET_META_DATA)
+            .metaData
+            .getBoolean(AUT_INIT_MANIFEST_KEY, true)
```

Comment:
I believe until developers didn't write `auto-init` and set it to `true` or `false`, we **Shouldn't** initialize library.

So with this definition, `defaultValue` of this function should be `false`.